### PR TITLE
Create `src/metrics/mod.rs` and move `src/cpu.rs` into `src/metrics/cpu.rs`.

### PR DESCRIPTION
**Pull request by Krout0n**
I want to start working on something other than metrics, so it's better to put the metrics in a deeper module. If you approve with this PR policy, I'll do the others.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -70,8 +70,14 @@ impl Agent {
         loop {
             let (tx, rx) = channel();
             interval.tick().await;
+            // TODO: Quit using Values, then use metrics::HostMetric.
```
Comment (ne-sachirou) [inline src/lib.rs:73]: 👍

File: src/lib.rs [hunk 2]
```diff
@@ -118,10 +124,10 @@ impl Agent {
 pub mod config;
 pub mod host_meta;
 
-mod cpu;
 mod disk;
 mod filesystem;
 mod interface;
 mod loadavg;
 mod memory;
+mod metrics;
```
Comment (ne-sachirou) [inline src/lib.rs]: 🙅‍♀️  Plaese use `metric`

**Comment by ne-sachirou** [comment]
The main purpose - to move some metric modules to a separate one - seems to be good.

**Comment by Krout0n** [comment]
I'm gonna rebase commits after getting approved.

**Review by ne-sachirou — APPROVED** [review]
🙆‍♀️

**Comment by Krout0n** [comment]
Thank you.

**Pull request merged**